sdcm/cluster_k8s/mini_k8s.py

```diff
@@ -403,10 +422,11 @@ def scylla_image(self):
         docker_repo = self.params.get('docker_image')
         scylla_version = self.params.get('scylla_version')
         if not scylla_version or not docker_repo:
-            return
+            return ""
         return f"{docker_repo}:{scylla_version}"
 
     def deploy(self):
+        self.setup_prerequisites()
```

Comment:
We don't need to do that if docker is already installed.

Reply:
Disagree.
Docker, minikube and kind do not guarantee that we have what is done in the prerequisites at all.
And my case is the example of it. I had docker and minikube installed before I used functional tests. And, obviously, I didn't set up lots of other stuff such as "conntrack" package or sysctl with higher value and so on...